indexer/src/main/java/au/org/aodn/esindexer/service/StacCollectionMapperService.java

```diff
@@ -618,7 +618,7 @@ else if (LinkUtils.isWfs(protocol)) {
                                             linkModel.setRel(RelationType.WFS.getValue());
                                         }
                                         else {
-                                            linkModel.setRel(RelationType.RELATED.getValue());
+                                            linkModel.setRel(protocol);
```

Comment:
Please have a check in the LinkUtils, I found that we can map some of these standard stuff to DATA, change the DATA_FILE to DATA

    private final static List<String> DATA = Arrays.asList(
            "WWW:DOWNLOAD-1.0-http--download",
            "WWW:DOWNLOAD-1.0-http--downloadother",
            "WWW:DOWNLOAD-1.0-http--downloaddata",
            "WWW:DOWNLOAD-1.0-ftp--download",
            "WWW:LINK-1.0-http--downloaddata"
    );

    private final static List<String> RELATED = Arrays.asList(
            "WWW:LINK-1.0-http--link",
            "WWW:LINK-1.0-http--related",
    );

    private final static List<String> METADATA = List.of(
            "WWW:LINK-1.0-http--metadata-URL"
    );

then check and map it accordingly